<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1011</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>60</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>处理内容：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>282</x>
        <y>470</y>
        <width>60</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>处理进度：</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>720</width>
        <height>435</height>
       </rect>
      </property>
      <property name="title">
       <string>检测情况</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="detailsWidget">
         <property name="minimumSize">
          <size>
           <width>700</width>
           <height>400</height>
          </size>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>序号</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>桩号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>负责人</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>日期</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>药柱(个数)</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>下药杆(个数)</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>设计井深</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>雷管个数</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>药柱个数</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>下药杆数目</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>人工复检</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>状态</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="processDetails">
      <property name="geometry">
       <rect>
        <x>76</x>
        <y>470</y>
        <width>200</width>
        <height>18</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>无</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>348</x>
        <y>471</y>
        <width>112</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>20</y>
        <width>131</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="title">
            <string>编辑</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="loadButton">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>加载数据</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>:/images/resources/images/pg1.ico</normaloff>:/images/resources/images/pg1.ico</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="saveAllButton">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>全部保存</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="addButton">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>添加条目</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteButton">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>删除条目</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="saveButton">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>保存当前</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="saveFileButton">
                <property name="text">
                 <string>导出文件</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>统计</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="updateStatusButton">
                <property name="text">
                 <string>更新状态</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>导入文件</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>预留4</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>931</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>检测</string>
      </property>
      <widget class="QTableWidget" name="videoWidget_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>881</width>
         <height>191</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>选择</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Video Files</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="startDetectionButton">
       <property name="geometry">
        <rect>
         <x>830</x>
         <y>310</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>开始检测</string>
       </property>
      </widget>
      <widget class="QPushButton" name="startDetectionButton_2">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>310</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>人工复查</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>19</y>
         <width>881</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>工区路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filelineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="openFileButton">
          <property name="text">
           <string>open file</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>330</y>
         <width>181</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="processEdit">
          <property name="text">
           <string>检测进度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="detectProgressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QProgressBar" name="totaldetectProgressBar">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>330</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>931</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>检测结果</string>
      </property>
      <widget class="QTextBrowser" name="resultText">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>881</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableWidget" name="testWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>831</width>
        <height>451</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>选择</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>视频名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>桩号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>设计钻井深度</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>设计激发深度 </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>单井药量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>单井雷管数量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>井数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>下药深度</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>差值</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>井监</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>检查人</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>下药日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>检查日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>钻井结果</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>备注</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>811</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>条件查询</string>
      </property>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>232</width>
          <height>23</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>桩号</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="stationnumberlineEdit"/>
        <widget class="QPushButton" name="stationnumberpushButton">
         <property name="text">
          <string>执行</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>151</width>
        <height>571</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>531</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>工区</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuweishenem1">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="action_Excel"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_D"/>
    <addaction name="actionWatch"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_detectors"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuweishenem1"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="action_create"/>
   <addaction name="action_open_workspace"/>
   <addaction name="action_D"/>
   <addaction name="actionWatch"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>退出系统(E)</string>
   </property>
   <property name="toolTip">
    <string>退出系统</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开文件(O)</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionWatch">
   <property name="text">
    <string>查看检测情况（W）</string>
   </property>
   <property name="toolTip">
    <string>查看检测信息</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>开始检测（D）</string>
   </property>
   <property name="toolTip">
    <string>开始检测</string>
   </property>
  </action>
  <action name="action_Excel">
   <property name="text">
    <string>查看Excel表格</string>
   </property>
   <property name="toolTip">
    <string>查看</string>
   </property>
  </action>
  <action name="action_create">
   <property name="text">
    <string>创建工区（C）</string>
   </property>
   <property name="toolTip">
    <string>创建工区</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_open_workspace">
   <property name="text">
    <string>打开工区</string>
   </property>
  </action>
  <action name="action_detectors">
   <property name="text">
    <string>检测器设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
